app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,62 @@
+class DashboardsController < ApplicationController
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  layout 'dashboard'
+  before_action :authenticate_user!
+
+  def index
+    @page_title = 'Dashboard'
+  end
+
+  def new_user
+    authorize :dashboard, :new_user?
+    @page_title = 'Dashboard -> Add User'
+    @user = User.new({})
```

Comment:
is User.new({}) and User.new() perform the same action?

Reply:
In my learning depth, there is no difference between them, but I used **robocop**. It suggested User.new({}) instead of User.new(), and that's why I used it. I removed it. 